datarobot_batch_scoring/main.py

```diff
@@ -255,6 +255,7 @@ def parse_generic_options(parsed_args):
     compression = parsed_args['compress']
     out_file = parsed_args['out']
     timeout = int(parsed_args['timeout'])
+    timeout = None if timeout == -1 else timeout
```

Comment:
It seems weird to specify the default as -1 just to be a sentinel for "don't do a timeout". Why not set the default to be timeout and then change 257 to be
```
timeout = parsed_args['timeout']
timeout = None if timeout is None else int(timeout)
```

It just seems like it would be less surprising to look at the defaults and have it actually be the defautls.

Reply:
Hmmm. I didn't think of that alternative. I just saw that it generated an error so I went for the easy fix.

Reply:
Someone clever made sure that you can't have defaults of None:
```
    parsed_args = {k: v
                   for k, v in vars(parser.parse_args(argv)).items()
                   if v is not None}
    return parsed_args
```